recipes/sentry-native/all/conandata.yml

```diff
@@ -1,7 +1,7 @@
 sources:
-  "0.7.10":
-    url: "https://github.com/getsentry/sentry-native/releases/download/0.7.10/sentry-native.zip"
-    sha256: "b7f7b5002cf7a4c614736ac294351da499db4f7fe155a452d527e69badc672bc"
+  "0.7.11":
```

Comment:
Please, keep 0.7.10 as well. You can open a separated PR in case wants to clean the recipe by removing old versions.

Reply:
oops my bad